Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM ubuntu:23.04
 WORKDIR /bpftime
 RUN apt-get update && apt-get install -y --no-install-recommends \
         libelf1 libelf-dev zlib1g-dev make cmake git libboost1.74-all-dev \
```

Comment:
Was boost installed here?

Reply:
Yes, install libboost-all-dev here so that the next steps can run successfully

Reply:
I mean that a package named libboost1.74-all-dev was already installed, doesn't this work?